### PR TITLE
Added filter to control css output.

### PR DESCRIPTION
**Pull request by marksabbath**
Addresses #56

### CODE REVIEW

**Comment by marksabbath** [comment]
Hey, @nickcernis !

Would you mind taking a look at this PR?

Thanks in advance :)

File: simple-social-icons.php [hunk 1]
```diff
@@ -58,6 +58,13 @@ class Simple_Social_Icons_Widget extends WP_Widget {
 	 */
 	protected $active_instances;
 
+	/**
+	 * Constrols custom css output.
```
Comment (nickcernis) [inline simple-social-icons.php]: Looks like a typo?

Constrols → Controls

File: simple-social-icons.php [hunk 2]
```diff
@@ -199,6 +206,13 @@ function __construct() {
 			),
 		) );
 
+		/**
+		 * If setting this filter to true, we recommend mentioning in your theme documentation
+		 * that Simple Social Icons widget settings will noy display styling options, as your
```
Comment (nickcernis) [inline simple-social-icons.php]: Another: noy → not

**Comment by nickcernis** [comment]
@marksabbath A couple of typos and one suggestion for the `form` function, but otherwise looks good!

File: simple-social-icons.php (hunk 1 above)
Comment (marksabbath) [reply simple-social-icons.php]: What can I say about that? 🤦‍♂️

File: simple-social-icons.php (hunk 2 above)
Comment (marksabbath) [reply simple-social-icons.php]: What can I say about that? 🤦‍♂️ version 2!

Comment (marksabbath) [reply simple-social-icons.php]: Resolved by the last commit.

File: simple-social-icons.php (hunk 1 above)
Comment (marksabbath) [reply simple-social-icons.php]: Should be fixed by the last commit.

**Comment by marksabbath** [comment]
Should be all fixed @nickcernis thanks for all the help!

**Pull request merged**